canopy/canopyjs/src/protobuf/index.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import protobuf, { Root } from 'protobufjs';
+import protoJSON from '../compiled_protos.json';
+
+const root: Root = protobuf.Root.fromJSON(protoJSON);
+
+export default Object.keys(root)
```

Comment:
Is this actually needed since the `compiled_protos.js` file gets generated?

Reply:
no, it is not. Updated